public/stylesheets/style.css

```diff
@@ -167,3 +167,11 @@ h6 {
   color: rgba(255, 255, 255, 0.6);
 }
 
+.no-margins {
+  margin: 0;
+}
+
+.no-borders {
+  border: 0 !important;
+  border-radius: 0 !important;
+}
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/public/stylesheets/style.css
+++ b/public/stylesheets/style.css
@@ -174,4 +174,4 @@
 .no-borders {
   border: 0 !important;
   border-radius: 0 !important;
-}+}
```